code_style.sh

```diff
@@ -61,12 +61,11 @@ fi
 
 # format JS
 cd "${root}/web" || exit 1
-yarn_args='--silent --no-progress --non-interactive'
 yarn_lint_args="$([ -n "$fix" ] && echo 'lint:fix' || echo 'lint')"
-if ! yarn $yarn_args install; then
+if ! yarn install; then
```

Comment:
You're using `yarn` here and `npx yarn` in CMake. Is this correct or should there be a single way of executing it? Asking as I'm not sure how to adjust the CI bots configuration.

Reply:
I think I'm the one who introduced `yarn` into the mix because it's what I use when developing, but it's just a personal preference rather than a requirement.

If it's causing headaches then let's just use npm instead of yarn in the build. (e.g. `yarn`, `npx yarn` -> `npm`, removing yarn.lock, adding the npm lockfile instead)

Reply:
I don't have a preference as I'm not doing any web development myself. The confusion was around the difference in the way the command is invoked.

Reply:
Understood; I should've been clearer that my comment was for @FluxState

Reply:
Using yarn is a good step forward. npm/npx are not necessary at all. I have added those commands only because build/test machines do not have NodeJS 16. On the 16 LTS the only thing you need to do is `corepack enable` to install/enable yarn. With earlier versions on those various systems it was a try-fail-and-try-again to find a combination of commands that would actually install and run yarn.

The main requirement is NodeJS 16 LTS and `corepack enable`. The second one can be run by the web build but it feels like it belongs to the system configuration phase.